src/unix/lwt_unix.h

```diff
@@ -24,20 +24,20 @@
 
 #define CAML_NAME_SPACE
 
-#include <lwt_config.h>
 #include <caml/mlvalues.h>
-#include <caml/unixsupport.h>
 #include <caml/socketaddr.h>
+#include <caml/unixsupport.h>
+#include <lwt_config.h>
```

Comment:
Argh, [this build failure](https://ci.appveyor.com/project/aantron/lwt/build/515/job/itjy9yrwatj5m9p7#L446) suggests that either `caml/unixsupport.h` or `lwt_config.h` has to manually be moved above `caml/socketaddr.h`. This is an upstream issue due to one of the headers not including the other one internally, but we have to work around it :/